keras_core/applications/applications_test.py

```diff
@@ -107,6 +107,10 @@ def _get_elephant(target_size):
     os.environ.get("SKIP_APPLICATIONS_TESTS"),
     reason="Env variable set to skip.",
 )
+@pytest.mark.skipif(
```

Comment:
This is probably worth making a pytest fixture for.

Then we can just say @pytest.mark.requires_trainable_backend
(You can see an example of how to set this up in `conftest.py` [here](https://github.com/keras-team/keras-cv/blob/master/keras_cv/conftest.py#L47)

Reply:
+1